geoapps/peak_finder/driver.py

```diff
@@ -331,6 +331,5 @@ def rotation_2D(angle):
 if __name__ == "__main__":
     file = sys.argv[1]
     params = PeakFinderParams(InputFile.read_ui_json(file))
-    params.geoh5.close()
```

Comment:
I think we have to close it here, otherwise the run will try to re-open in r+

Reply:
That was the case until we made the read_ui_json method close the file.